recipes/serialio/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Where did this license file come from? I can't find it in the upstream repo.

Reply:
Hi @dopplershift, 

Thanks for the review, I copied the file according to the license from: https://www.gnu.org/licenses/gpl-3.0.txt.

Reply:
Is that based on the metadata in the `setup.py` [upstream](https://github.com/tiagocoutinho/serialio/blob/3282292f800599cdadc7ca57ee2cbef4037beb63/setup.py#L24-L27)? I'm not sure that enough for us legally to redistribute the code.

@ocefpaf @beckermr Am I off-base here?

Reply:
Upstream doesn't have a license file and we should not add one for them here. Can you ask them to do it? The license per se is fine and, if you want to submit a PR upstream, we can merge this as soon as it is accepted there.

Reply:
I created the PR to add the license file.